.github/index-schema.json

```
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "repo": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string",
+      "maxLength": 200
+    },
+    "type": {
+      "type": "string",
+      "enum": ["lib", "app"]
```

Another option could be data repositories.

```suggestion
      "enum": ["lib", "app", "data"]
```

---

Check #3 if you haven't seen it yet for more suggestions on the build keyword

---

BTW feel free to modify the PR as you see fit before merging.
I considered "adapter" because of all the coupling software out there for OpenFOAM to different things
These libs usually require an external repo to be fetched/compiled
ex. https://github.com/Crompulence/CPL_APP_OPENFOAM

Also,you may want a license entry too in the scheme